lib/location.js

```diff
@@ -261,10 +270,19 @@ var LocationState = State.extend(function (self, name, opts) {
         return '\n' + number + '. ' + address.label;
     };
 
-    self.generate_footer = function() {
+    if (self.skip_text !== null)
+    {
+        self.generate_footer = function() {
+        return '\n' + 'n. '+self.next_text + '\n' + 'p. '+self.previous_text + '\n' + 's. '+self.skip_text;
+        };
+    }
+    else
+    {
+        self.generate_footer = function() {
         return '\n' + 'n. '+self.next_text + '\n' + 'p. '+self.previous_text;
-
-    };
+        };
+    }
```

Comment:
I would avoid have separate definitions of `generate_footer` and just do something like: